packages/skill-api/package.json

```diff
@@ -33,27 +33,28 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.16.3",
-    "groq": "^2.15.0",
     "@sanity/client": "^3.4.1",
-    "@skillrecordings/database": "workspace:*",
     "@skillrecordings/commerce-server": "workspace:*",
     "@skillrecordings/convertkit-react-ui": "workspace:*",
+    "@skillrecordings/database": "workspace:*",
     "@skillrecordings/inngest": "workspace:*",
     "@slack/web-api": "^6.7.2",
     "concurrently": "^7.2.2",
     "cookie": "^0.5.0",
     "date-fns": "^2.28.0",
+    "groq": "^2.15.0",
     "handlebars": "^4.7.7",
     "html-to-text": "^8.2.0",
     "lodash": "^4.17.21",
-    "mjml": "^4.12.0",
     "micro": "^9.3.4",
+    "mjml": "^4.12.0",
     "node-html-markdown": "^1.2.0",
+    "pluralize": "^8.0.0",
     "preact": "^10.7.2",
     "preact-render-to-string": "^5.2.0",
     "sanitize-html": "^2.7.0",
     "uuid": "^8.3.2",
-    "pluralize": "^8.0.0"
+    "zod": "^3.21.2"
```

Comment:
Adding `Zod` to this package. Looks like pnpm rearranged some of the deps while doing that.